src/canvas-extractor/README.md

```diff
@@ -1,16 +1,41 @@
 # Canvas Extractor
 
-This script retrieves and writes out to CSV all students, active sections, assignments,
-and submissions.
+This tool retrieves and writes out to CSV students, active sections,
+assignments, and submissions by querying the Canvas API. For more information on
+the this tool and its output files, please see the main repository
+[readme](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit).
 
-## Requirements
+## Getting Started
 
-Requires Python 3.8 and [Poetry](https://python-poetry.org/). To get started,
-run the following command in the package directory:
+1. Ensure you have [Python 3.8+ and
+   Poetry](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit#getting-started)
+1. At a command prompt, install all required dependencies:
```

Comment:
I think it could be valuable for non-technical users to specify the directory where they should be placed before executing the commands.

Reply:
Thinking about it. Honestly, I can't imagine that many non-technical users would even try this. However, this ticket _is_ supposed to be for non-technical users.

Reply:
done